examples/wallet/src/util.ts

```diff
@@ -34,7 +34,6 @@ import {
     nodePort,
     walletProxyBaseUrl,
 } from './constants';
-import { filterRecord, mapRecord } from '../../../packages/sdk/lib/esm/util';
```

Comment:
Wow... that also seems like a bit of a hack 😲

Reply:
yes, probably some auto-completion was the cause of this